### PR TITLE
fix: Increase item pickup range

### PR DESCRIPTION
**Pull request by 4Denthusiast**
The code for extending item pickup range beyond the character collider was already present, but was broken accidentally in [#3907](https://github.com/MovingBlocks/Terasology/pull/3907/files/d8689f1710fb567d6ddfa3f2f9e6d4d09aedd768#diff-65b44f3586a2bf7259ca4594f2521535), so this requires very little change.

### CODE REVIEW

**Comment by kaen** [comment]
Confirmed this makes picking up the small rocks and branches in JS much easier :+1:

File: engine/src/main/resources/assets/prefabs/player/player.prefab [hunk 1]
```diff
@@ -20,6 +20,11 @@
   },
   "VisualCharacter": {
   },
+  // the CapsuleShape is used to detect item entities for pickup. The actual character collider is generated programmatically.
```
Comment (skaldarnar) [inline engine/src/main/resources/assets/prefabs/player/player.prefab:23]: Hm, I'm wondering whether this additional shape can be derived from the collider (to keep them in sync).
If it is a "special" shape just for item pickup, maybe this should be part of some better-named component?

Comment (skaldarnar) [reply engine/src/main/resources/assets/prefabs/player/player.prefab:23]: Additional question: where is the character collider shape programmatically set? I see that in https://github.com/MovingBlocks/Terasology/blob/develop/engine/src/main/java/org/terasology/physics/bullet/BulletPhysics.java#L491-L505 the character collider is created with values taken from the `CharacterMovementComponent`.

How can we keep a single source of truth for the (base) values?

Comment (4Denthusiast) [reply engine/src/main/resources/assets/prefabs/player/player.prefab]: I've now moved the pickup-radius information to the CharacterMovementComponent instead, with the same default value, which I think resolves this. There should be no difference in behaviour, but this makes it simpler for other things to alter the pickup radius later.

**Pull request merged**